recipe/meta.yaml

```
@@ -101,8 +102,9 @@ requirements:
     - qt6-charts
     - qt6-datavis3d
     - llvmdev
-    - clangdev
+    - clangdev {{ cxx_compiler_version }}
     - libclang
+    - libclang {{ cxx_compiler_version }}  # [osx]
```

The pins only make sense for osx (on linux, `cxx_compiler_version` has nothing to do with llvm).

```suggestion
    - llvmdev
    # for libclang
    - clangdev
    - llvmdev  {{ cxx_compiler_version }}  # [osx]
    - clangdev {{ cxx_compiler_version }}  # [osx]
```